ci/contract.hbs

```diff
@@ -0,0 +1,78 @@
+## `{{name}}`
+
+{{{natspec.userdoc}}}
+
+{{{natspec.devdoc}}}
+
+{{#if functions}}
+### Functions
+
+{{#each functions}}
+{{#if visibility }}
+```
+{{name}}({{args}}){{#if outputs}} → {{outputs}}{{/if}} ({{visibility}})
+```
+{{/if}}
+{{{natspec.userdoc}}}
+
+{{{natspec.devdoc}}}
+
+{{#if natspec.params}}
+Parameters:
```

Comment:
I think this should be a 4th level heading. Consider making all the `Parameters` equal to `#### Parameters:`

Reply:
+1